website/addons/wiki/templates/wiki_widget.mako

```diff
@@ -12,9 +12,8 @@
 <% import json %>
 <script>
     window.contextVars = $.extend(true, {}, window.contextVars, {
+        wikiWidget: true,
         usePythonRender: ${json.dumps(use_python_render)},
-        urls: {
```

Comment:
Because it was the only URL attached to that object on the dashboard page. I can revert that if it's better for the object to be there for future use